src/DIPS.Xamarin.UI/Controls/Sheet/SheetBehavior.cs

```diff
@@ -555,7 +581,7 @@ private async void ToggleSheetVisibility()
                 OnBeforeCloseCommand?.Execute(OnBeforeCloseCommandParameter);
                 OnBeforeClose?.Invoke(this, EventArgs.Empty);
 
-                m_modalityLayout.Hide(m_sheetView.SheetFrame);
+                m_modalityLayout.Hide(m_sheetView.SheetFrame, 600);

```

Comment:
600 for all cases? Or 250 without Anticipation?

Reply:
250 without.

Reply:
Fix'd